### PR TITLE
Remove instruction metrics (except max align)

### PR DESCRIPTION
**Pull request by gumb0**
Depends on #403

### CODE REVIEW

File: lib/fizzy/parser_expr.cpp [hunk 1]
```diff
@@ -248,7 +248,6 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, FuncIdx f
         std::tie(opcode, pos) = parse_byte(pos, end);
 
         auto& frame = control_stack.top();
-        const auto& metrics = metrics_table[opcode];
         const auto& type = type_table[opcode];
```
Comment (axic) [inline lib/fizzy/parser_expr.cpp:251]: I'd rather have `max_align` here.

Comment (gumb0) [reply lib/fizzy/parser_expr.cpp:251]: Why? It's used only in subset of instructions, which are handled all in one switch case, so why put it into wider scope?

Comment (axic) [reply lib/fizzy/parser_expr.cpp:251]: My reasoning is that none of the cases in the switch use the `opcode` variable and one must go up to the top to find it.

Comment (gumb0) [reply lib/fizzy/parser_expr.cpp:251]: Tried to benchmark, it doesn't make any difference.

Comment (axic) [reply lib/fizzy/parser_expr.cpp:251]: If there's no speed difference I'd prefer it at the top.

Comment (gumb0) [reply lib/fizzy/parser_expr.cpp:251]: Made `max_align` variable.

**Pull request merged**